docs/src/pages/index.soy

```diff
@@ -118,8 +118,8 @@ description: "A collection of well known Algebraic Data Types for your utter enj
       </section>
       <section class="highlight row">
         <div class="col-md-6 col-md-offset-2">
-          <h4 class="highlight-title">Combine a list of values down to one value</h4>
-          <p class="highlight-description">Crocks provides a large variety of Monoids, which are ADTs that represent a binary operation and are usually locked down it to a specific type. These are great when you need to combine two things into one under a specific operation.</p>
+          <h4 class="highlight-title">Combine values like never before</h4>
```

Comment:
These bullet points are going to change to highlight why to use crocks over other solutions, like:
* No external dependencies 
* Common interface between all the types
* Everything in one place.
* Pointfree and Fluent style friendly.

So no need to change these as they will change to provide reasons like above in the near fututre.